components/layout/construct.rs

```diff
@@ -408,6 +409,7 @@ impl<'a, ConcreteThreadSafeLayoutNode: ThreadSafeLayoutNode>
             Some(LayoutNodeType::Element(LayoutElementType::HTMLObjectElement)) => {
                 let image_info = Box::new(ImageFragmentInfo::new(
                     node.object_data(),
+                    node.image_density(),
```

Comment:
This will break because the node is not an image element. We should pass None here.